BridgeCareApp/BridgeCare/BridgeCareContext.cs

```diff
@@ -44,5 +52,19 @@ protected override void OnModelCreating(DbModelBuilder modelBuilder)
                 .Property(e => e.BUDGETORDER)
                 .IsUnicode(false);
         }
+
+        public DataTable NonEntitySQLQuery(string selectStatement)
```

Comment:
Consider moving it in a different class. BridgeCareContext class is usually only for entity mapping. You can create a separate class in DataAccessLayer